bestbook/api/books.py

```diff
@@ -43,17 +46,59 @@ class Topic(core.Base):
                      nullable=False)
     modified = Column(DateTime(timezone=False), default=None)
 
+    @classmethod
+    def upsert(cls, topic):        
+        if isinstance(topic, cls):
+            return topic
+        try:
+            return cls.get(name=topic)
+        except:
```

Comment:
Bare except is a dangerous habit that we really should break.  As discussed in [PEP8](https://www.python.org/dev/peps/pep-0008/#programming-recommendations) and https://realpython.com/the-most-diabolical-python-antipattern this can make bugs really hard to find because it sweeps so much under the rug.